--- node-18.13.0/configure.py.orig	2023-01-05 13:33:29.000000000 +0000
+++ node-18.13.0/configure.py	2023-02-14 15:03:06.138170340 +0000
@@ -1284,7 +1284,7 @@
 
   # Enable branch protection for arm64
   if target_arch == 'arm64':
-    o['cflags']+=['-msign-return-address=all']
+    # o['cflags']+=['-msign-return-address=all'] # this is unsupported
     o['variables']['arm_fpu'] = options.arm_fpu or 'neon'
 
   if options.node_snapshot_main is not None:
