--- node-18.14.0/deps/v8/src/execution/ppc/simulator-ppc.cc.orig	2023-02-01 20:54:58.000000000 +0000
+++ node-18.14.0/deps/v8/src/execution/ppc/simulator-ppc.cc	2023-02-16 09:01:52.897950047 +0000
@@ -28,6 +28,16 @@
 #include "src/runtime/runtime-utils.h"
 #include "src/utils/ostreams.h"
 
+template <typename... Args>
+auto std::ceilf(Args&&... args) -> decltype(::ceilf(std::forward<Args>(args)...)) {
+  return ::ceilf(std::forward<Args>(args)...);
+}
+
+template <typename... Args>
+auto std::floorf(Args&&... args) -> decltype(::floorf(std::forward<Args>(args)...)) {
+  return ::floorf(std::forward<Args>(args)...);
+}
+
 // Only build the simulator if not compiling for real PPC hardware.
 namespace v8 {
 namespace internal {
