From 8c9e1ba2c766827abcb7192c4c3703be717e8da1 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 12 Aug 2024 14:41:41 +0200
Subject: [PATCH] Use C++17

---
 common.gypi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/common.gypi b/common.gypi
index d713ad55..a71eb997 100644
--- a/common.gypi
+++ b/common.gypi
@@ -402,7 +402,7 @@
       }],
       [ 'OS in "linux freebsd openbsd solaris android aix cloudabi"', {
         'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
-        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++14' ],
+        'cflags_cc': [ '-fno-rtti', '-fno-exceptions', '-std=gnu++17' ],
         'defines': [ '__STDC_FORMAT_MACROS' ],
         'ldflags': [ '-rdynamic' ],
         'target_conditions': [
@@ -542,7 +542,7 @@
           ['clang==1', {
             'xcode_settings': {
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
-              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++14',  # -std=gnu++14
+              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++17',  # -std=gnu++14
               'CLANG_CXX_LIBRARY': 'libc++',
             },
           }],
