--- node-18.14.0/deps/v8/src/inspector/v8-runtime-agent-impl.cc.orig	2023-02-01 20:54:58.000000000 +0000
+++ node-18.14.0/deps/v8/src/inspector/v8-runtime-agent-impl.cc	2023-02-16 09:01:37.247765494 +0000
@@ -30,7 +30,7 @@
 
 #include "src/inspector/v8-runtime-agent-impl.h"
 
-#include <inttypes.h>
+#include <cinttypes>
 
 #include "../../third_party/inspector_protocol/crdtp/json.h"
 #include "include/v8-container.h"
