From 9398c7b7b01b467a046ceb9ea5a8ef924eb49ab5 Mon Sep 17 00:00:00 2001
From: Mario Rodas <marsam@users.noreply.github.com>
Date: Thu, 4 Jul 2019 04:20:00 -0500
Subject: [PATCH] build: link libatomic only in clang on linux (https://github.com/nodejs/node/pull/28532)

---
 node.gyp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/node.gyp b/node.gyp
index 9a5556c2eff4..d20d469dc670 100644
--- a/node.gyp
+++ b/node.gyp
@@ -289,7 +289,7 @@
           '-Wl,-bnoerrmsg',
         ],
       }],
-      ['(OS=="linux" or OS=="mac") and llvm_version!=0', {
+      ['OS=="linux" and llvm_version!=0', {
         'libraries': ['-latomic'],
       }],
     ],
